sc2reader/utils.py

```diff
@@ -20,15 +20,18 @@ class DepotFile(object):
     """
 
     #: The url template for all DepotFiles
-    url_template = "https://{0}-s2-depot.classic.blizzard.com/{1}.{2}"
+    url_template = "https://{}-s2-depot.classic.blizzard.com{}/{}.{}"
 
     def __init__(self, bytes):
         #: The server the file is hosted on
         self.server = bytes[4:8].decode("utf-8").strip("\x00 ")
-
+        #: Used to make it possible to load maps from CN. This isn't needed for any other region and so is blank by default.
```

Comment:
```suggestion
        # Used to make it possible to load maps from CN. This isn't needed for
        # any other region and so is blank by default.
```
psf/black will fail any lines that are longer than 88 characters.

Reply:
Probably also line 45 below

Reply:
Evidently it wasnt a fan of trailing spaces.